### PR TITLE
Set up server

### PR DESCRIPTION
**Pull request by xueccc**

### CODE REVIEW

**Comment by xueccc** [comment]
@ChuaJ Would you please re-review. I changed the id from app to shopProducts.

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,8 +1,23 @@
-var component = function() {
-  var element = document.createElement('div');
-  element.innerHTML = _.join(['Shop', 'Prodcuts'], ' ');
-  return element;
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+var axios = require('axios');
```
Comment (ChuaJ) [inline client/src/index.jsx]: Sorry I was unclear! You only need one or the other (axios or jquery) to make your network requests. But whichever one you choose should be in the form of an import, not a require.

Comment (xueccc) [reply client/src/index.jsx]: Cool, thanks Jesse. I plan to use bootstrap, will need jquery for styling. I will keep both for now.

**Comment by xueccc** [comment]
@ChuaJ I changed the syntax from require to import, would you please take a look. See commit 
4b49597.

**Pull request merged**